### PR TITLE
Basic support for custom themes

### PR DESCRIPTION
**Pull request by rsletta**
Prevent eyes from burning from the white background by implementing basic color scheme for new Alv branding 
Use Mudblazor components for login flow

### CODE REVIEW

File: packages/adminpanel/Client/Shared/MainLayout.razor [hunk 1]
```diff
@@ -25,8 +25,60 @@
...
+        },
+        Typography = new Typography()
+        {
+            Default = new Default()
+            {
+                FontFamily = ["League Spartan", "sans-serif"]
+            },
+            H1 = new H1()
+            {
+                FontFamily = ["Libre Baskerville", "sans-serif"]
```
Comment (palbock) [inline packages/adminpanel/Client/Shared/MainLayout.razor]: Kan vi lagre denne FontFamily'en i en variabel?

Comment (rsletta) [reply packages/adminpanel/Client/Shared/MainLayout.razor]: Ja, meget godt poeng. Nå er du god.

File: packages/adminpanel/Client/Shared/MainLayout.razor [hunk 2]
```diff
@@ -24,9 +24,70 @@
 
 @code {
     bool _drawerOpen = true;
+    
+    /* Branding */
```
Comment (rsletta) [inline packages/adminpanel/Client/Shared/MainLayout.razor:28]: @palbock, er dette rett format?

File: packages/adminpanel/Client/Shared/MainLayout.razor [hunk 3]
```diff
@@ -24,9 +24,70 @@
...
+    private const string AlvBrandingBlue = "#243F4D";
+    private const string AlvBrandingGreen = "#9DAC86";
+    private const string AlvBrandingLightBrown = "#E6C08D";
+    private const string AlvBrandingBeige = "#F8F1E3";
+    
+    private static readonly string[] AlvBrandFontFamilyHeading = ["Libre Baskerville", "sans-serif"];
+    private static readonly string[] AlvBrandFontFamilyDefault = ["League Spartan", "sans-serif"];
```
Comment (rsletta) [inline packages/adminpanel/Client/Shared/MainLayout.razor:35]: @palbock, er dette rett format?

**Review by palbock — APPROVED** [review]

**Pull request merged**